eZ/Publish/Core/Repository/Values/User/UserReference.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * File containing the eZ\Publish\Core\Repository\Values\User\User class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\Repository\Values\User;
+
+use eZ\Publish\API\Repository\Values\User\UserReference as APIUserReference;
+
+/**
+ * This class represents a user reference for use in sessions and Repository.
+ */
+class UserReference implements APIUserReference
+{
+    /**
+     * @var mixed
+     */
+    private $id;
```

Comment:
Please rename to `$userId` to be consistent with the getter (same for argument in constructor).